hasura/metadata/databases/default/tables/public_user.yaml

```diff
@@ -48,8 +50,9 @@ select_permissions:
         - is_subscribed
         - name
       filter:
-        id:
-          _eq: X-Hasura-User-Id
+        memberships:
```

Comment:
is there a way we don't expose auth0Id and is_subscribed other than to yourself?

Reply:
Not sure about this with the single `user` hasura role.

Reply:
ok let's track this in a Linear issue to revisit

Reply:
[WID-657](https://linear.app/worldcoin/issue/WID-657/make-auth0id-and-is-subscribed-fields-visible-only-for-yourself)